dotcom-rendering/src/components/Discussion.tsx

```diff
@@ -70,7 +70,7 @@ const commentIdFromUrl = () => {
 	return parseInt(commentId, 10);
 };
 
-const remapFilters = (
+const remapToValidFilters = (
```

Comment:
Do we strictly need this renaming as part of this PR?

Reply:
Fair point, I'll extract to a separate pr